### ISSUE TITLE
[SvelteKit] `isLoading` last worked in v2.1.26, broken since v2.1.28

### ISSUE THREAD

**Issue by pheuter** [vercel-labs/ai#416]
```
"@sveltejs/kit": "1.22.4",
"ai": "2.1.31",
"openai-edge": "1.2.2"
```

```svelte
$: ({ messages, input, isLoading, handleSubmit, setMessages } = useChat({
	api: `/ai/conversations/${conversationId}`,
	experimental_onFunctionCall: async (chatMessages) => {
		return {
			messages: chatMessages
		};
	}
}));

$: console.log($isLoading); // Always logs `false`
```

The code above always logs `false`, even when the request is clearly loading. If I pin the version of `ai` to 2.1.26 it works again.

**Comment by MaxLeiter** [comment]
Thanks for the report, I know what's causing this and am taking a look :+1:

**Comment by MaxLeiter** [comment]
The fix is included in [ai@2.1.32](https://github.com/vercel-labs/ai/releases/tag/ai%402.1.32)

**Comment by lgrammel** [comment]
## Identify issue type

Identified as **Bug**  
Confidence: **High**

### Rationale
This is a confirmed regression in existing SvelteKit `useChat` loading-state behavior that required a product fix.

### Evidence
- `useChat`'s `isLoading` store always remained `false` during active requests in `ai` 2.1.31.
- The reporter identified a regression: the behavior worked in 2.1.26 and broke starting in 2.1.28.
- A maintainer confirmed the cause and stated that a fix was released in `ai@2.1.32`.
